Documentation/reference/user/10_filesystem.rst

```diff
@@ -458,12 +458,11 @@ are two conditions where ``mmap()`` can be supported:
 1. ``mmap()`` can be used to support *eXecute In Place* (XIP) on random
```

Comment:
This commit looks good, although the ordering is a bit silly. I expect this needs to be merged before any shmfs stuff is merged.